path_config.go

```diff
@@ -266,6 +280,12 @@ func (b *jwtAuthBackend) pathConfigWrite(ctx context.Context, req *logical.Reque
 		return logical.ErrorResponse("invalid response_mode: %q", config.OIDCResponseMode), nil
 	}
 
+	// Validate provider_config
+	_, err := NewProviderConfig(config, ProviderMap)
+	if err != nil {
+		return logical.ErrorResponse("invalid provider_config: %s", err), nil
+	}
```

Comment:
Because we ignore the returned value here and only care about the error, can we re-write this to [reduce the scoping of variables][reduce-scope]:

```go
if _, err := NewProviderConfig(config, ProviderMap); err != nil {
  return logical.ErrorResponse("invalid provider_config: %s", err), nil
}
```


[reduce-scope]: https://github.com/uber-go/guide/blob/master/style.md#reduce-scope-of-variables

Reply:
Yep, changed in fe083e8